src/coord/src/catalog/storage.rs

```diff
@@ -92,6 +92,11 @@ const MIGRATIONS: &[&str] = &[
         name TEXT PRIMARY KEY,
         value TEXT
     );",
+    // Makes the mz_internal schema literal so it can store functions.
+    //
+    // Introduced in v0.6.2.
+    "INSERT INTO schemas VALUES
+        (4, NULL, 'mz_internal');",
```

Comment:
Ah, crap, this isn't gonna work for existing clusters which might have a different schema with ID 4. I think we'll just have to let this ID float around. `INSERT INTO schemas (database_id, name) VALUES (NULL, 'mz_internal')` should work to let SQLite pick an available schema ID.